src/App.vue

```diff
@@ -14,6 +14,8 @@ import Menu from '@/components/Menu.vue';
 import { loadingController } from '@ionic/vue';
 import emitter from "@/event-bus";
 import { useStore } from '@/store'
+import { mapGetters } from 'vuex';
```

Comment:
Use a single import from vuex for useStore and mapGetters

Reply:
improved sir